### ISSUE TITLE
JSON is not valid

### ISSUE THREAD

**Issue by Dron-Z** [esnet/iperf#613]
I,m using iperf3 in test VoIP channels.
Iperf3 command:
```shell
iperf3 -c 192.168.66.132 -u -t10 -p 6969 -l 250 -b 67k
```
In a few time, iperf3 return no valid JSON, part JSON output below
```json
{
                        "streams":      [{
                                        "socket":       4,
                                        "start":        2.00027,
                                        "end":  3.00018,
                                        "seconds":      0.99991,
                                        "bytes":        8250,
                                        "bits_per_second":      66005.9,
                                        "jitter_ms":    6.11273e+08,
                                        "lost_packets": 0,
                                        "packets":      0,
                                        "lost_percent": -nan,
                                        "omitted":      false
                                }],
                        "sum":  {
                                "start":        2.00027,
                                "end":  3.00018,
                                "seconds":      0.99991,
                                "bytes":        8250,
                                "bits_per_second":      66005.9,
                                "jitter_ms":    6.11273e+08,
                                "lost_packets": 0,
                                "packets":      0,
                                "lost_percent": -nan,
                                "omitted":      false
                        }
}

**Comment by bmah888** [comment]
What version of iperf3 exhibited this problem?  This should have been fixed in #278 (or rather, the first part of #278).  There were several other bugs that were duped to that as well.  Bottom line, iperf-3.2 or most 3.1.x releases shouldn't have this problem.

**Comment by Dron-Z** [comment]
I'm updated the iperf. Problem solved. Thanks
